app/risks.py

```diff
@@ -34,10 +36,19 @@
 with open(Path(__file__).parent.joinpath('data', 'departements.geojson'), 'rb') as response:
     departments = json.load(response)
 
+# We fetch the deparment risk score json and store it in the risk_json variable
+# risk_json = requests.get("http://pyro-risks.herokuapp.com/risk/fr/2020-09-01").json()
```

Comment:
Is this line useful ? If yes in for which scenario ?

Reply:
It was my suggestion: this is the line to use once the timeout will be taken care of on the risk prediction server :ok_hand: 
But on 2nd thought, we can add this later on and avoid displaying the URL publicly! So perhaps we should remove this in the end

Reply:
OK I deleted the line, we'll add it later